.github/workflows/build.yml

```diff
@@ -7,6 +7,8 @@ on:
     branches: [master]
     tags:
       - v*
+  schedule:
+    - cron: '18 1 * * 4'
```

Comment:
Could you add a comment like `At 01:18 on Thursday`?

Reply:
Done